### DISCUSSION TITLE
Roll ups and governor limit of 100

### DISCUSSION THREAD

**Discussion by coopermorrison** [SFDO-Community/declarative-lookup-rollup-summaries#1140] [Q&A]
Hi,

We have reached the governor limit of 100 scheduled jobs within our salesforce org. Most of these are from rollups to summarised various business data.

Salesforce support has recommended implementing a schedulable class and that class would invoke two batch jobs one after the other one. Is there a way to invoke the DLRS batch jobs through this, I would e.g. run all Account related roll-ups in one schedulable class.

Currently stumped - any advice or suggestions would be greatly appreciated.

`//declaration of scheduler class

global class ScheduleMultipleBatchJobs implements schedulable {

//method to implement Apex schedulers

global void execute(SchedulableContext ct)

{

BatchDeletion BD = new BatchDeletion();

Database.executebatch(BD);

UpdateAccountFields UP = new UpdateAccountFields();

Database.executebatch(UP);

}

}`

**Comment by aheber** [comment]
As of right now there isn't a way to chain rollup batch jobs. It is on our roadmap but I couldn't give you any idea of when it might be done. There are several things ahead of it right now.
